src/main/java/com/github/fishio/control/HelpScreenController.java

```diff
@@ -60,6 +64,10 @@ private void displayKeyBinding(KeyCode key, int y) {
 		map.put(KeyCode.DOWN, new Vec2d(1418, 415));
 		map.put(KeyCode.LEFT, new Vec2d(1334, 415));
 		map.put(KeyCode.RIGHT, new Vec2d(1501, 415));
+		map.put(KeyCode.W, new Vec2d(228, 202));
+		map.put(KeyCode.S, new Vec2d(249, 275));
+		map.put(KeyCode.A, new Vec2d(169, 275));
+		map.put(KeyCode.D, new Vec2d(334, 275));
 		//TODO map more keys when allowing key changes
```

Comment:
We should either use CSS styling to create the wanted effect, or create an image where keys are sorted from a-z etc., but this is something for later.

<feature